UKHomeOffice/cop-ui
# COP-10181: Use new cop form endpoint

### Description
When loading new style COP (React) forms, it should go to the new endpoint (identified by a "cop-" prefix on the form name); otherwise, it should go to the existing endpoint for FormBuilder/Form.io forms.

### To test
Go to the Forms list, open Developer Tools and switch to the Network tab. If you select any of the existing forms, you should see it tries to source the form from `/form/name/{ form_name }`. If you choose any of the new-style forms, you should see it tries to source the form from `/copform/name/{ form_name }`. The differentiator is the name of the form - the new-style ones all begin with `'cop-'`.
